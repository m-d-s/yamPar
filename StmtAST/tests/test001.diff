1d0
< Empty
