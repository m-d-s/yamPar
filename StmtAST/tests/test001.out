Call:   Top
  Call:   Stmt
    Call:   Empty
      Consumed token: <";" at line 1 column 1>
    Return: Empty
  Return: Stmt
  Consumed token: <<EOF> at line 1 column 26>
Return: Top
Empty
